### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by catdematos98**
displays list of events in user profile

### CODE REVIEW

**Comment by jennhe** [comment]
YUH

**Review by jennhe — COMMENTED** [review]
YUH

**Review by jennhe — APPROVED** [review]
looks good

File: app/src/main/java/com/example/teamproject/UserProfileFragment.java [hunk 1]
```diff
@@ -36,6 +68,113 @@ public void onClick(View v) {
...
+    private void getAttendingEvents() {
+        attendingEvents = new ArrayList<Ad>();
+        final Ad.Query adQuery = new Ad.Query();
+        adQuery.withUser();
+        adQuery.whereContains("rsvp", currentUser.getObjectId());
+        adQuery.include("rsvp");
+        adQuery.whereEqualTo("rsvp", ParseObject.createWithoutData(ParseUser.class, currentUser.getObjectId()));
+        adQuery.findInBackground(new FindCallback<Ad>() {
+            @Override
+            public void done(List<Ad> objects, ParseException e) {
```
Comment (baldwinchang) [inline app/src/main/java/com/example/teamproject/UserProfileFragment.java:140]: Indentation is weird

**Review by baldwinchang — COMMENTED** [review]
Looks good to me. I would like to see more technical depth in the coming week. (code design that is further past the scope of CodePath)

**Pull request merged**